### ISSUE TITLE
MolMeDb

### ISSUE THREAD

**Issue by tabbassidaloii** [BioDataFuse/pyBiodatafuse#145]
@YojanaGadiya  The query is based on UniProt ID meaning there might be multiple ids for each gene id. but the output table reports only one UniProt id per gene.
How do we decide which id?

**Comment by YojanaGadiya** [comment]
Shouldn't there just be a single Swiss-Prot id for each protein in a species?

**Comment by tabbassidaloii** [comment]
That is true, but each gene may have multiple isoforms and therefore multiple protein id (UniProt)

**Comment by YojanaGadiya** [comment]
Fixed with #146 and PR #147
